crud/upsert.lua

```diff
@@ -104,4 +104,47 @@ function upsert.call(space_name, obj, user_operations, opts)
     }
 end
 
+--- Update or insert an object in the specified space
+--
+-- @function object
+--
+-- @param string space_name
+--  A space name
+--
+-- @param table obj
+--  Object (according to space format)
```

Comment:
`Object`?

Reply:
Fixed in https://github.com/tarantool/crud/compare/57d8d662f964250f6720c850a734721e730144ba..0583a496d5d16035d34ddad4447cd1ca862b8788